### ISSUE TITLE
Predicting the GNSS Pseudo-Measurement with a Hybrid Multi-Head Attention for Tightly-Coupled Navigation

### ISSUE THREAD

**Issue by weisongwen** [weisongwen/researchTools#188]
The demand for high-accuracy continuous positioning of vehicles in complex environments has become more and more urgent. Although existing tightly-coupled navigation based on GNSS/INS can provide continuous positioning in these scenes with bad GNSS signals or short-term GNSS outages, it still cannot meet the application requirements. To tackle the aforementioned problems, we propose a novel algorithm to predict GNSS pseudo-measurement when the GNSS signals are blocked. The algorithm introduces a hybrid multi-head attention neural network to learn the dynamic nonlinear relationship of the raw GNSS observations. Three attention neural networks are used to capture richer features from the pseudorange, pseudorange rate, satellite position and carrier position as well as the satellite velocity and the carrier velocity, respectively, which can improve the generalization ability and avoid overfitting. Extensive experiments on dataset show that the proposed algorithm can obtain at least
